custom_components/tahoma/executor.py

```diff
@@ -1,6 +1,8 @@
 """Class for helpers and community with the OverKiz API."""
+from __future__ import annotations
+
 import logging
-from typing import Any, Optional
```

Comment:
Optional does not exist? I don't get it

Reply:
Optional wasn't part of my original change, the main issue is that `list` is supported in Python 3.9, where it is called `List` in Python 3.8. 

After importing from future, it seems that importing Optional is not required anymore.